source/MRViewer/MRPalette.h

```diff
@@ -68,7 +68,8 @@ class Palette
     /// Discrete: bar consists of single colored rectangles for each initial color
     /// Linear (default): color is changing from one to another during initial color list
     /// \param onlyTopHalf if true, draws only top half of the palette and labels stretched to whole window
-    MRVIEWER_API void draw( ImDrawList* drawList, float scaling, const ImVec2& pos, const ImVec2& size, bool onlyTopHalf = false ) const;
+    /// \param labelBackgroundFromViewport true - grab the color from the active viewport, false - from ColorTheme (getBackgroundColor_)
+    MRVIEWER_API void draw( ImDrawList* drawList, float scaling, const ImVec2& pos, const ImVec2& size, bool onlyTopHalf = false, bool labelBackgroundFromViewport = true ) const;
```

Comment:
please add new
```
void Palette::draw( ImDrawList* drawList, float scaling, const ImVec2& pos, const ImVec2& size, bool onlyTopHalf, const Color& bgColor ) const
```
and keep old
```
void Palette::draw( ImDrawList* drawList, float scaling, const ImVec2& pos, const ImVec2& size, bool onlyTopHalf ) const
```
implemented via new

Reply:
reworked